scripts/kcl/manager/be/README.md

```
@@ -0,0 +1,219 @@
+
+Run the test files to verify functionality:
+
+```bash
+kcl run be/function_registry_test.k
+kcl run be/pipeline_test.k
+kcl run be/crossplane_function_test.k
+```
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)